### PR TITLE
Undo selection

### PR DESCRIPTION
**Pull request by CPKreuz**

### CODE REVIEW

File: PixiEditor/Helpers/SelectionHelpers.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+using PixiEditor.Models.DataHolders;
+using PixiEditor.Models.Enums;
+using PixiEditor.Models.Position;
+using PixiEditor.Models.Undo;
+
+namespace PixiEditor.Helpers
+{
+    public class SelectionHelpers
+    {
+        public static void SelectionSet(object[] arguments)
```
Comment (flabbet) [inline PixiEditor/Helpers/SelectionHelpers.cs:11]: Change the name to SetSelectionProcess, generally keep the names for undo methods using OriginalName + Process

**Review by flabbet — CHANGES_REQUESTED** [review]
Perfect, I love how the code is clean. One thing I would change is not to deselect on undo. Only remove selection when user wants to (deselect manually or via Undo). You didn't add this, it was before, but because we have undo now, you can remove that behavior.

**Review by flabbet — CHANGES_REQUESTED** [review]
Adding and subtracting selection should be undoable

**Comment by CPKreuz** [comment]
> Adding and subtracting selection should be undoable

Well it is, it's just treated as a new one, will fix

**Review by flabbet — APPROVED** [review]
Fabulous

**Pull request merged**